docs/src/main/asciidoc/security-authorize-web-endpoints-reference.adoc

```diff
@@ -465,9 +471,37 @@ By default, string-based permission is performed by the `io.quarkus.security.Str
 <5> Permissions are not beans, therefore only way to obtain bean instances is programmatically via the `Arc.container()`.
 
 CAUTION: If you plan to use the `@PermissionsAllowed` on the IO thread, review the information in xref:security-proactive-authentication-concept.adoc[Proactive Authentication].
+
 NOTE: The `@PermissionsAllowed` is not repeatable on class-level due to limitations of Quarkus interceptors.
 Please find well-argued explanation in the xref:cdi-reference.adoc#repeatable-interceptor-bindings[Repeatable interceptor bindings] section of the CDI reference.
 
+Provided you are already using role-based authorization, the easiest way of adding permissions to `SecurityIdentity` is permission to role mapping.
```

Comment:
```suggestion
Provided `SecurityIdentity` contains roles, the easiest way to add permissions to `SecurityIdentity` is to create a roles to permissions mapping.
```

Reply:
I'm not sure we can talk about `role-based authorization` in this context since the purpose is to use `permission-based authorization`, so something similar to what is suggested (please change the wprding of you'd like) can be clearer IMHO

Reply:
Actually, let me clarify something else first

Reply:
OK, IMHO the proposed update still makes sens, since enforcing RBAC is orthogonal (or complementary) to the permission based control